src/TritonCTS/src/DbWrapper.cpp

```diff
@@ -259,6 +265,8 @@ void DbWrapper::writeClockNetsToDb(const Clock& clockNet) {
         }
 
         std::cout << "    Created " << numClkNets << " clock nets.\n";
+        unsigned currentTotalNets = _options->getNumClockSubnets() + numClkNets;
```

Comment:
once upon a time I thought unsigned was a good way to document values that cannot be negative.
the first time you do a subtraction and end up with a negative value you will realize this is a bad idea.